gardening/data/project_task_type.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="project_task_type_2" model="project.task.type">
+    <field name="name">Inbox</field>
+  </record>
+  <record id="project_task_type_3" model="project.task.type">
+    <field name="sequence">2</field>
+    <field name="name">Today</field>
+  </record>
+  <record id="project_task_type_4" model="project.task.type">
+    <field name="sequence">3</field>
+    <field name="name">This Week</field>
+  </record>
+  <record id="project_task_type_5" model="project.task.type">
+    <field name="sequence">4</field>
+    <field name="name">This Month</field>
+  </record>
+  <record id="project_task_type_6" model="project.task.type">
+    <field name="sequence">5</field>
+    <field name="name">Later</field>
+  </record>
+  <record id="project_task_type_7" model="project.task.type">
+    <field name="sequence">6</field>
+    <field name="name">Done</field>
+    <field name="fold" eval="True"/>
+  </record>
+  <record id="project_task_type_8" model="project.task.type">
+    <field name="sequence">7</field>
+    <field name="name">Cancelled</field>
+    <field name="fold" eval="True"/>
+  </record>
+  <record id="project_task_type_14" model="project.task.type">
+    <field name="sequence" eval="False"/>
+    <field name="name">Clean up</field>
+  </record>
+  <record id="project_task_type_15" model="project.task.type">
+    <field name="name">Soil preparation</field>
+  </record>
+  <record id="project_task_type_16" model="project.task.type">
+    <field name="sequence">2</field>
+    <field name="name">Repairs</field>
+  </record>
+  <record id="project_task_type_17" model="project.task.type">
+    <field name="sequence">3</field>
+    <field name="name">Planting</field>
+  </record>
+  <record id="project_task_type_18" model="project.task.type">
+    <field name="sequence">4</field>
+    <field name="name">Final Steps</field>
+  </record>
+  <record id="project_task_type_19" model="project.task.type">
+    <field name="sequence">5</field>
+    <field name="name">Done</field>
+  </record>
+</odoo>
```

Comment:
```suggestion
```
seems useless

Reply:
Some are standard, indeed, but some look new, so probably useful

Reply:
Why?

Reply:
none of them are used

Reply:
Yes 14->19 are used in project_project